webroot/results/admin/persons_finish_unfinished_ACTION.php

```diff
@@ -125,6 +141,30 @@ function insertPerson( $newName, $newCountry, $newId ) {
   
   #--- Execute the command.
   dbCommand( $command );
+
+  if ( $oldCountry <> $newCountry || $oldName <> $newName){
+
+    #--- Guess countries iso2
+    $newCountryIso2 = getIso2FromCountryId( $newCountry );
+    if ( $newCountryIso2 ){
+      if ( $oldCountry == $newCountry ){
+        $oldCountryIso2 = $newCountryIso2;
+      } else {
+        $oldCountryIso2 = getIso2FromCountryId( $oldCountry );
+      }
+      if ( $oldCountryIso2 ){
+        #--- Build the command.
+        $command = "
+          UPDATE InboxPersons SET name='$newName', countryId='$newCountryIso2'
```

Comment:
I was a bit surprised to see iso2 being used here, because I thought we store our weird custom country ids in the database, not ISO2 strings. Is the answer that we store iso2 in InboxPersons, but then when we create the actual row in Persons later on, we convert it from iso2 to a country id?